system/global.nix

```diff
@@ -47,6 +47,22 @@
     fuse-overlayfs
   ];
 
+  programs.rust-motd = {
+    enable = true;
+    enableMotdInSSHD = true;
+    settings = {
+      banner = {
+        color = "red";
+        command = ''
+          ${pkgs.nettools}/bin/hostname | ${pkgs.figlet}/bin/figlet -f slant
+          systemctl --failed --quiet
```

Comment:
What does `--quit` do? I would assume this is not showing any messages from the `--failed` command, am I wrong? If the failed services are displayed, then LGTM.